From 8c08c98e0b63163de9b73669abaf15dcb47e5c97 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Tue, 29 Mar 2016 09:14:52 -0400
Subject: [PATCH] Require include files from the default location

---
 bin/lessc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/bin/lessc b/bin/lessc
index 4907bc36f62e427e39903ca47be0c0c3ba613d13..8c484539a2e06b4b21af7b235d397fbb66bcbbdf 100755
--- a/bin/lessc
+++ b/bin/lessc
@@ -1,20 +1,20 @@
 #!/usr/bin/env node
 
 var path = require('path'),
-    fs = require('../lib/less-node/fs'),
+    fs = require('less-node/fs'),
     os = require("os"),
     errno,
     mkdirp;
 
 try {
     errno = require('errno');
 } catch (err) {
     errno = null;
 }
 
-var less = require('../lib/less-node'),
+var less = require('less-node'),
     pluginLoader = new less.PluginLoader(less),
     plugin,
     plugins = [];
 
 var args = process.argv.slice(1);
@@ -483,6 +483,6 @@ function printUsage() {
 
         process.stdin.on('end', function() {
             parseLessFile(false, buffer);
         });
     }
-})();
\ No newline at end of file
+})();
-- 
2.7.3

