From 6135de47cbfbbb9bcebe3df7a8b2418b9f0f36fa Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Wed, 19 Dec 2012 09:06:24 -0500
Subject: [PATCH 1/2] Require include files from the default location

---
 bin/lessc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/bin/lessc b/bin/lessc
index 809741d870030ae4f177788315e11d35885b60ae..4b2471bfc5d7048aee43d8364af0cc6770056605 100755
--- a/bin/lessc
+++ b/bin/lessc
@@ -1,12 +1,12 @@
 #!/usr/bin/env node
 
 var path = require('path'),
-    fs = require('../lib/less/fs'),
+    fs = require('less/fs'),
     sys = require('util'),
     os = require('os'),
     mkdirp;
 
-var less = require('../lib/less');
+var less = require('less');
 var args = process.argv.slice(1);
 var options = {
     depends: false,
@@ -97,7 +97,7 @@ args = args.filter(function (arg) {
             break;
         case 'h':
         case 'help':
-            require('../lib/less/lessc_helper').printUsage();
+            require('less/lessc_helper').printUsage();
             continueProcessing = false;
         case 'x':
         case 'compress':
@@ -249,7 +249,7 @@ args = args.filter(function (arg) {
             }
             break;
         default:
-            require('../lib/less/lessc_helper').printUsage();
+            require('less/lessc_helper').printUsage();
             continueProcessing = false;
             currentErrorcode = 1;
             break;
@@ -294,7 +294,7 @@ if (options.cleancss && options.sourceMap) {
 if (! input) {
     console.log("lessc: no input files");
     console.log("");
-    require('../lib/less/lessc_helper').printUsage();
+    require('less/lessc_helper').printUsage();
     currentErrorcode = 1;
     return;
 }
-- 
1.9.3

