From a318f959688a23603e2868568f4489edaa100f86 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Thu, 4 Sep 2014 10:14:33 -0400
Subject: [PATCH 2/2] Fedora: disable sourcemap tests

---
 test/index.js | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/test/index.js b/test/index.js
index 33e79637f05dca0cade43fccb69eddc739584ea6..ed2d4c391b6e605eafdc384960195d3824387a6b 100644
--- a/test/index.js
+++ b/test/index.js
@@ -29,6 +29,7 @@ lessTester.runTestSet({strictMath: true, dumpLineNumbers: 'all'}, "debug/", null
 lessTester.runTestSet({strictMath: true, relativeUrls: false, rootpath: "folder (1)/"}, "static-urls/");
 lessTester.runTestSet({strictMath: true, compress: true}, "compression/");
 lessTester.runTestSet({}, "legacy/");
+/*
 lessTester.runTestSet({strictMath: true, strictUnits: true, sourceMap: true, globalVars: true }, "sourcemaps/",
     lessTester.testSourcemap, null, null, 
     function(filename, type) { 
@@ -37,6 +38,8 @@ lessTester.runTestSet({strictMath: true, strictUnits: true, sourceMap: true, glo
         }
         return path.join('test/sourcemaps', filename) + '.json'; 
     });
+
+*/
 lessTester.runTestSet({globalVars: true, banner: "/**\n  * Test\n  */\n"}, "globalVars/",
     null, null, null, function(name) { return path.join('test/less/', name) + '.json'; });
 lessTester.runTestSet({modifyVars: true}, "modifyVars/",
-- 
2.1.0

